test/units/testsuite-07.mount-invalid-chars.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+set -eux
+set -o pipefail
+
+# Don't send invalid characters over dbus if a mount contains them
+
+at_exit() {
+    mountpoint -q /proc/1/mountinfo && umount /proc/1/mountinfo
+    [[ -e /tmp/fstab.bak ]] && mv -f /tmp/fstab /etc/fstab
```

Comment:
The source file for `mv` is missing an extension.

Reply:
Oops, addressed in https://github.com/systemd/systemd/pull/27733.